libs/sdk-ui-dashboard/api/sdk-ui-dashboard.api.md

```diff
@@ -1607,6 +1607,13 @@ export interface DashboardContext {
     workspace: string;
 }
 
+// @beta
+export interface DashboardContextObject {
```

Comment:
Naming is a bit confusing, as we already have DashboardContext in the dashboard component.

Reply:
renamed, thanks